### PR TITLE
[Bug 18086] Update LCB style guide.

### PR DESCRIPTION
**Pull request by peter-b**
- Naming types
- Wrapping at 80 columns
- Indentating continuation lines
- Writing `OnLoad()` handlers
- Basic property/event guidelines

### CODE REVIEW

File: config/ios.gypi [hunk 1]
```diff
@@ -28,8 +28,8 @@
 		'SHARED_PRECOMPS_DIR': '$(OBJROOT)/Precompiled/$(CURRENT_ARCH)',
 		'GCC_VERSION': '',
 		'ALWAYS_SEARCH_USER_PATHS': 'NO',
-		'IPHONEOS_DEPLOYMENT_TARGET[arch=armv7]': '5.1.1',
```
Comment (montegoulding) [inline config/ios.gypi:31]: @peter-b is this meant to be in this patch?

Comment (peter-b) [reply config/ios.gypi:31]: @montegoulding Nope! Will fix.

**Comment by peter-b** [comment]
@montegoulding Okay, I've removed the erroneous iOS configuration changes.

**Comment by montegoulding** [comment]
@livecode-vulcan review ok 9cb994a

**Comment by livecode-vulcan** [comment]
💙 review by @montegoulding ok 9cb994aeddc13878931a1ef24dff22db9988abc3

**Comment by livecode-vulcan** [comment]
😎 test success 9cb994aeddc13878931a1ef24dff22db9988abc3
- try-community-linux-x86: [success](https://vulcan.livecode.com/builders/try-community-linux-x86/builds/191)
- try-community-mac: [success](https://vulcan.livecode.com/builders/try-community-mac/builds/195)
- try-community-linux-x86_64: [success](https://vulcan.livecode.com/builders/try-community-linux-x86_64/builds/191)
- try-community-emscripten: [success](https://vulcan.livecode.com/builders/try-community-emscripten/builds/191)
- try-community-ios: [success](https://vulcan.livecode.com/builders/try-community-ios/builds/195)
- try-community-win-x86: [success](https://vulcan.livecode.com/builders/try-community-win-x86/builds/191)
- try-community-android-armv6: [success](https://vulcan.livecode.com/builders/try-community-android-armv6/builds/191)

**Comment by peter-b** [comment]
@livecode-vulcan retry 9cb994a

**Comment by livecode-vulcan** [comment]
😎 test success 9cb994aeddc13878931a1ef24dff22db9988abc3
- try-community-linux-x86: [success](https://vulcan.livecode.com/builders/try-community-linux-x86/builds/229)
- try-community-mac: [success](https://vulcan.livecode.com/builders/try-community-mac/builds/233)
- try-community-linux-x86_64: [success](https://vulcan.livecode.com/builders/try-community-linux-x86_64/builds/229)
- try-community-emscripten: [success](https://vulcan.livecode.com/builders/try-community-emscripten/builds/229)
- try-community-ios: [success](https://vulcan.livecode.com/builders/try-community-ios/builds/233)
- try-community-win-x86: [success](https://vulcan.livecode.com/builders/try-community-win-x86/builds/229)
- try-community-android-armv6: [success](https://vulcan.livecode.com/builders/try-community-android-armv6/builds/229)

**Pull request merged**